Resources/Prototypes/Entities/Objects/Misc/translator_implanters.yml

```
@@ -75,3 +75,11 @@
   components:
   - type: Implanter
     implant: MofficTranslatorImplant
+
+- type: entity
+  id: ValyrianStandardTranslatorImplanter
+  parent: [ BaseTranslatorImplanter ]
```

yamllint

[error] too many spaces inside brackets

(brackets)